packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -651,8 +652,14 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         return IdentifierWhitelistInterface(finder.getImplementationAddress(OracleInterfaces.IdentifierWhitelist));
     }
 
-    // Stamps the ancillary data blob with the optimistic oracle tag denoting what contract requested it.
-    function _stampAncillaryData(bytes memory ancillaryData, address requester) internal pure returns (bytes memory) {
-        return abi.encodePacked(ancillaryData, "OptimisticOracle", requester);
+    function _appendToAncillaryData(bytes memory ancillaryData, address requester)
+        internal
+        pure
+        returns (bytes memory)
+    {
+        // Since this contract will be the one to formally submit DVM price requests, its useful for voters to know who
+        // the original requester was.
+        bytes memory prefix = ",requester:";
```

Comment:
How do we deal with commas? Some callers could pass in empty ancillary data -- is our format okay with an extra leading comma? Should we add a conditional? Or should we do something else?

Not super opinionated, just wanted to raise the issue.

Reply:
We could check length of `bytes`:

```
        bytes memory prefix;
        if (ancillaryData.length > 0) {
            prefix = ",requester:";
        } else {
            prefix = "requester:";
        }
        return abi.encodePacked(ancillaryData, prefix, AncillaryData.toUtf8Bytes(requester));
```

Reply:
Yep, that seems like a good solution!